main.go

```diff
@@ -49,17 +56,21 @@ const (
 func init() {
 	/** CLI flags */
 	// Max number of repositories to fetch from registry (default = 5)
-	repoCount = flag.Int("repos", 5, "number of repositories to garbage collect")
+	repoCount = flag.Int("count", 5, "number of repositories to garbage collect")
 	// Base URL of registry (default = http://localhost:5000)
 	registryURL = flag.String("registry", "http://localhost:5000", "URL of registry")
+	// registry username (default = "")
+	username = flag.String("username", "", "registry username to login")
+	// registry password (default = "")
+	password = flag.String("password", "", "registry password to login")
```

Comment:
I'm not fond of Plaintext passwords as args, we should do this in an extra step (just as `mysql -u root -p` does)